### PR TITLE
Keep code quality high with shellcheck

### PR DESCRIPTION
**Pull request by szepeviktor**
See https://github.com/koalaman/shellcheck/wiki/TravisCI

### CODE REVIEW

**Comment by szepeviktor** [comment]
Please consider fixing these shellcheck warnings

https://travis-ci.org/lukas2511/dehydrated/builds/168084127#L269

File: test.sh [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # Fail early
 set -eu -o pipefail
 
+NGROK_URL="https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip"
```
Comment (leonklingele) [inline test.sh]: Why was this changed?

Comment (leonklingele) [inline test.sh]: (I'm referring to the URL)

**Comment by szepeviktor** [comment]
See https://ngrok.com/download


-- 
+36204242498
Ezen a készüléken sok az elütés. 
Elnézést!

File: test.sh (hunk 1 above)
Comment (mrwacky42) [inline test.sh]: That's where the [ngrok Download](https://ngrok.com/download) page points. [Equinox.io](https://equinox.io/) seems to be some sort of Go project hosting.

**Comment by lukas2511** [comment]
I agree that shellcheck is useful and cleaning the code should be done, but I don't want any tiny false-positive resulting in tests failing, so I won't merge this into the test environment.
